okhttp/src/main/java/okhttp3/internal/http/HttpEngine.java

```diff
@@ -731,6 +731,10 @@ private Response readNetworkResponse() throws IOException {
       networkResponse = networkResponse.newBuilder()
           .body(httpStream.openResponseBody(networkResponse))
           .build();
+    } else if (networkResponse.code() != 101) {
```

Comment:
This block has the same contents as the one above it, so just merge the conditionals with an `||`.